test/studies/590o/telmas/dpll.chpl

```diff
@@ -29,8 +29,8 @@ config const printTiming = true;
 var numVars : int(32); // number of variables
 var numClss : sync int(32); // number of clauses
 
-var domVars : domain(1) dmapped (Block); // domain for variables
-var domClss : domain(1) dmapped (Block); // domain for clauses
+var domVars : domain(1) dmapped (blockDist); // domain for variables
```

Comment:
See my previous comments about the`dmapped(blockDist)`. However, seeing it in this form with the space makes more sense to me.

Reply:
This one's also not tested (as were the others between my first explanation and this one).